indico/modules/events/features/util.py

```diff
@@ -26,7 +27,7 @@ def get_feature_definition(name):
     try:
         return get_feature_definitions()[name]
     except KeyError:
-        raise RuntimeError(f'Feature does not exist: {name}')
+        raise RuntimeError(_('Feature does not exist: {feature_name}.').format(feature_name=name))
```

Comment:
This one does not need i18n, it is an error only developers will see in case of broken code.

Reply:
Reverted that change